src/Models/Calendar.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace LaravelEnso\Calendar\App\Models;
+namespace LaravelEnso\Calendar\Models;
 
 use Illuminate\Database\Eloquent\Model;
-use LaravelEnso\Calendar\App\Contracts\Calendar as Contract;
-use LaravelEnso\Rememberable\App\Traits\Rememberable;
-use LaravelEnso\TrackWho\App\Traits\CreatedBy;
+use LaravelEnso\Calendar\Contracts\Calendar as Contract;
+use LaravelEnso\Rememberable\Traits\Rememberable;
+use LaravelEnso\TrackWho\Traits\CreatedBy;
```

Comment:
you forgot here and in reminders to set $guarded = ['id']. Make sure to double check request validators / controllers as well.